dts/arm/st/f1/stm32f105.dtsi

```diff
@@ -85,5 +85,24 @@
 				#pwm-cells = <3>;
 			};
 		};
+
+		usbotg_fs: usb@50000000 {
+			compatible = "st,stm32-otgfs";
+			reg = <0x50000000 0x40000>;
+			interrupts = <67 0>;
+			interrupt-names = "otgfs";
+			num-bidir-endpoints = <4>;
+			ram-size = <1280>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x00001000>;
```

Comment:
According to Reference Manual RM0008,
USB clock is on APB1 bit 23
```suggestion
			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00800000>;
```

Reply:
`USB-FS` / USB OTG is in `RCC_AHBENR` bit 12 (which I believe makes the original correct).

Bit 23 of `RCC_APB1ENR` is reserved for the STM32F105... note it's a connectivity line device (thus Chapter 8, not Chapter 7, and Chapter 28 / `OTG_FS`, not Chapter 23 / `USB`).

Reply:
Ok, your are right, my bad.